Xamarin.Forms.Platform.UAP/WindowsIsolatedStorage.cs

```diff
@@ -4,13 +4,15 @@
 using Windows.Storage;
 using Windows.Storage.FileProperties;
 using Windows.Storage.Streams;
-using Xamarin.Forms.Internals;
+//using FileMode = Xamarin.Forms.Internals.FileMode;
+//using FileAccess = Xamarin.Forms.Internals.FileAccess;
+//using FileShare = Xamarin.Forms.Internals.FileShare;
```

Comment:
why do we need commented out `using`s ?

Reply:
i m going to add this later we need to make UAP compile under multi target for pre fall's update.  But thats another task